docs/user_api/index.rst

```diff
@@ -35,5 +35,6 @@ Helper Functions
    helpers.calculate_spherical_triangle_jacobian
    helpers.calculate_spherical_triangle_jacobian_barycentric
    get_gauss_quadratureDG
+   helpers.grid_center_lat_lon
```

Comment:
@erogluorhan I just added this function for the user API. I was wondering if I should re-organize this list to be in alphabetical order and add `helpers.` in front of `get_gauss_quadrature` and `get_tri_quadrature`?

Reply:
Yes, please do so!

Reply:
You can also mention this change in the PR description for future reference.